en/components/themes/typography.md

```diff
@@ -6,149 +6,210 @@ _extraFont: https://fonts.googleapis.com/css?family=Titillium+Web:300,400,600,70
+
+### Component Typography
+
+Most of the components in Ignite UI for Angular use scale categories for styling the text. For instance, the `igx-card` component uses the following scale categories:
+- `h5` - used for styling card title.
+- `subtitle-2` - used for styling card subtitle and small title.
+- `body-2` - used for styling card text content.
+
+There are two ways to change the text styles of a card. The first is by modifying the `h5`, `subtitle-2`, and/or `body-2` scales in the ***default type scale*** that we pass to the typography mixin. So if we wanted to make the title in a card smaller, all we have to do is change the font-size for the `h5` scale category.
+
+```scss
+// Create a custom h5 scale category style
+$my-h5: igx-type-style($font-size: 18px);
+
+// Create a custom type scale with the modified h5
+$my-type-scale: igx-type-scale($h5: $my-h5);
+
+// Pass the custom scale to the global typography mixin
+@include igx-typography($type-scale: $my-type-scale);
+```
+
+Note, however, that the above code will modify the `h5` scale category globally, which will affect the look and feel of all components that use the `h5` scale. This is done for consistency so that all `h5` elements look the same across your app. We understand that you may want to apply the modification for `h5` to specific components only, like the `igx-card` component in our case. This is why every component has its own typography mixin, which accepts a type scale itself, as well as a category configuration.
+
+```scss
+// Create a custom h5 scale category style
+$my-h5: igx-type-style($font-size: 18px);
+
+// Create a custom type scale with the modified h5
+$my-type-scale: igx-type-scale($h5: $my-h5);
+
+// Pass the custom scale to the card typography mixin only
+@include igx-card-typography($type-scale: $my-type-scale);
+```
+
+We no longer include the `igx-typography` mixin by passing it the `$my-type-scale` scale with our modification to the `h5` category. Now all we do is pass the custom scale we created to the `igx-card-typography` mixin. The only component that uses our `$my-type-scale` scale is the card now.
+
+Typography style mixins can be scope to specific selectors. Say we wanted our custom card typography styles to be applied for all `igx-card` components with class name of `my-cool-card`. 
+
+```scss
+//...
+.my-cool-card {
+    @include igx-card-typography($type-scale: $my-type-scale);
+}
+```
+
+The typography component mixins take a second argument - `$categories`. It is used to configure which parts of the component use what typography scale category. For instance, if we wanted the our custom card to use a different scale category for the title than `h5`, we could change it.
```

Comment:
... if we wanted [the/our - choose one!] custom card...
Ex. "if we wanted our custom card"...